<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DiaDubl</class>
 <widget class="QDialog" name="DiaDubl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1440</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DiaDubl</string>
  </property>
  <widget class="QWidget" name="centralWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1460</width>
     <height>883</height>
    </rect>
   </property>
   <widget class="QLabel" name="la_Bild1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>700</width>
      <height>700</height>
     </rect>
    </property>
    <property name="text">
     <string>Bild 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="la_Bild2">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>90</y>
      <width>700</width>
      <height>700</height>
     </rect>
    </property>
    <property name="text">
     <string>Bild 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_Nxt">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>850</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;&gt;&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="la_B_Pfad">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>820</y>
      <width>611</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>850</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Verdachtsf√§lle:</string>
    </property>
   </widget>
   <widget class="QLabel" name="la_verd">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>850</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1061</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Quelle</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Direktory</string>
     </property>
    </widget>
    <widget class="QLabel" name="la_dir">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>671</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_Dir">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pb_Abbruch">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>880</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Abbruch</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
